beast/physicsmodel/stars/ezmist/simpletable.py

```diff
@@ -71,13 +72,13 @@
 if PY3:
     iteritems = operator.methodcaller('items')
     itervalues = operator.methodcaller('values')
-    basestring = (str, bytes)
+    str = (str, bytes)
```

Comment:
I'm not sure I understand this comment.  What particular change are you suggesting?